_includes/head.html

```diff
@@ -31,7 +34,7 @@
 <![endif]-->
 
 {% if site.head_scripts %}
-  {% for script in site.head_scripts %}
-    <script src="{{ script | relative_url }}"></script>
-  {% endfor %}
-{% endif %}
+{% for script in site.head_scripts %}
```

Comment:
I don't quite understand the flattening of indentation here.

Reply:
Good catch; this was a mistake due to the auto-linter I'm using being a little over-zealous. 👍 I had generally saved without formatting but must have missed it once or twice. Will revert those pieces.